apps/storybook-e2e/src/e2e/sections/header/vl-header.stories.cy.ts

```diff
@@ -18,4 +18,34 @@ describe('story vl-header', () => {
         cy.createStubForEvent('vl-header', 'ready');
         cy.get('@ready').should('have.been.calledOnce');
     });
+
+    describe('vl-header - container <#header__container>', () => {
+        it('should render', () => {
+            cy.visit(headerUrl);
+
+            cy.get('vl-header');
+            cy.get('#header__container').should('exist');
+        });
+
+        it('should render with some basic styles', () => {
```

Comment:
je kan het hier specifieker beschrijven,
bv. `should render with fixed height`

Reply:
opgelost